packages/storybook/stories/va-file-input-multiple-uswds.stories.tsx

```diff
@@ -427,12 +427,12 @@ CustomValidation.args = {
   hint: 'Select any file type',
 };
 
-const EncryptedTemplate = ({ label, name, hint }) => {
+const EncryptedTemplate = ({ label, name }) => {
   const [encryptedList, setEncryptedList] = useState([]);
 
   function setEncrpytedForEachFile(event) {
     const fileEntries = event.detail.state;
-    const pdfFiles = fileEntries.map((file, index) => {
```

Comment:
`hint` and `index` were removed from the encrypted template from this story. Was that because they just weren't being used?

Reply:
Yeah, unused